topostats/default_config.yaml

```diff
@@ -63,6 +63,7 @@ plotting:
   zrange: [null, null]  # low and high height range for core images (can take [null, null]). low <= high
   colorbar: true  # Options : true, false
   axes: true # Options : true, false (due to off being a bool when parsed)
+  num_ticks: null # Number of ticks to have along the xy axes. Options : null (auto) or integer
```

Comment:
Hey Neil, please could you explain this further? This is already overridable and in the config so I don't understand what you mean.

Reply:
Sorry for not being clearer.

In my view everything in the style sheet should also be configurable, thus the `font.size` and `axis.titlesize` should be in the configuration so that users outside of the group can change them.

Ultimately anyone can either extract the data they want to plot and do so manually or hack at the code.

Good software gives people freedom and doesn't impose constraints on them.